src/benchmark.cpp

```diff
@@ -1,6 +1,8 @@
 #include "core/benchmark.hpp"
 #include "core/types.hpp"
 
+#include <boost/test/included/unit_test.hpp>
+
```

Comment:
this include actually should be in the benchmark.hpp.

Reply:
I will test and change it, thanks!